src/features/posts/postsSlice.tsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-param-reassign */
```

Comment:
Please remove the 'no-param-reassign' ESLint rule disable comment. The '@reduxjs/toolkit' package allows for direct state manipulation and it internally takes care of immutability. This means that the rule is not required to be disabled.